core/src/com/mygdx/minigolf/model/components/PowerUpAffectable.java

```diff
@@ -0,0 +1,20 @@
+package com.mygdx.minigolf.model.components;
+
+import com.badlogic.ashley.core.Component;
+import com.mygdx.minigolf.model.Effect;
+
+import java.time.Duration;
+import java.util.Map;
+
+public class PowerUpAffectable implements Component {
+
+    Map<Effect, Duration> effect;
+
+    public Map<Effect, Duration> getEffect() {
+        return effect;
+    }
+
+    public void setEffect(Map<Effect, Duration> effect) {
+        this.effect = effect;
+    }
```

Comment:
Kan være det er bedre for performance å gjøre effect (altså mapen) final og så ha getere og setere som endrer innholdet i effects.